### ISSUE TITLE
battery contradiction

### ISSUE THREAD

**Issue by JohnDenker** [ericmazur/PnPbook#138]
Figure 26.16 in section 26.4 on page 694 makes it clear that negative ions flow left-to-right while positive ions flow right-to-left.

However, figure 26.17 on the facing pages shows negatively-charged bisulfate ions flowing right-to-left.

Would anybody care to explain this?

**Comment by ericmazur** [comment]
Yes, that’s right. They HAVE to flow in the opposite direction in to cause the battery to work. See, for example Purcell’s book. This flow in the opposite direction is due to the fact that near the electrodes there is a very steep potential gradient (the “seat of the potential” in Purcell’s words). It’s a good idea reading up on it.

Eric

**Comment by ericmazur** [comment]
From @JohnDenker: consider using a Li-ion battery

**Comment by JohnDenker** [comment]
Upon further consideration, I'm not sure that it's a good idea to drop the lead-acid battery in favor of the LI-ion battery.

The rules say we should consider **all** the data.  We should not select only the data that supports this-or-that pet theory.  I reckon it would be better to label figure 26.16 as an _idealized_ battery, and then explain that sometimes real batteries are significantly more complicated.  This illustrates a number of interesting points about science, and about how science is done.  Section 26.4 seems somewhat less than clear on these points:
-   Example ≠ proof.
-   Plausibility ≠ proof.
-   Black-box description ≠ microscopic explanation.
-   Idealized battery ≠ real battery.
-   Correct on average ≠ correct in detail.
-   Force ≠ motion.
-   Drift ≠ diffusion.
-   Entropy ≠ energy.

For details on what I mean by this, see
  https://www.av8n.com/physics/battery.htm#sec-perspective
